tests/unit/cmd/scan/test_path.py

```diff
@@ -557,3 +557,47 @@ def test_scan_context_repository(
             and arg.get("GGShield-Repository-URL") == "github.com/owner/repository"
             for arg in scan_mock.call_args[0]
         )
+
+    @patch("pygitguardian.GGClient.multi_content_scan")
+    @my_vcr.use_cassette("test_scan_context_repository.yaml")
+    def test_scan_path_with_fallback_repository_url(
+        self,
+        scan_mock: Mock,
+        tmp_path: Path,
+        cli_fs_runner: CliRunner,
+    ) -> None:
+        """
+        GIVEN a repository without a remote url
+        WHEN executing a scan with REPOSITORY_REMOTE_FALLBACK set
+        THEN the environment variable value is sent in the headers
+        """
+        local_repo = Repository.create(tmp_path)
+
+        file = local_repo.path / "file_secret"
+        write_text(file, "Hello")
+        local_repo.add(file)
+        local_repo.create_commit()
+
+        scan_result = MultiScanResult([])
+        scan_result.status_code = 200
+        scan_mock.return_value = scan_result
+
+        fallback_url = "https://github.com/fallback/repository.git"
+        with patch.dict(os.environ, {"REPOSITORY_REMOTE_FALLBACK": fallback_url}):
+            result = cli_fs_runner.invoke(
+                cli,
+                [
+                    "secret",
+                    "scan",
+                    "path",
+                    str(file),
+                ],
+            )
+        assert result.exit_code == ExitCode.SUCCESS, result.output
```

Comment:
You can use `assert_invoke_ok()` here:

```suggestion
        assert_invoke_ok(result)
```